test/Npgsql.Tests/Npgsql.Tests.csproj

```diff
@@ -1,10 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <PropertyGroup>
-    <TargetFrameworks>net451;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```

Comment:
As above, please revert to continue supporting net451 for now.

Reply:
Yikes, meant to be excluded from the commit. Removing.